Fixes some spelling errors

Index: dphys-swapfile-20061020/dphys-swapfile
===================================================================
--- dphys-swapfile-20061020.orig/dphys-swapfile	2010-05-05 21:02:12.000000000 +0200
+++ dphys-swapfile-20061020/dphys-swapfile	2010-05-05 21:02:55.000000000 +0200
@@ -32,7 +32,7 @@
 NAME=dphys-swapfile
 PNAME=dphys-swapfile
 
-# check user config file, let user overwride settings
+# check user config file, let user override settings
 #   swap file place/filename and size
 if [ -f /etc/"${PNAME}" ] ; then
   . /etc/"${PNAME}"
@@ -49,7 +49,7 @@
       echo -n "computing size, "
       # /proc/kcore became unreliable so let's use /proc/meminfo instead
       MEMSIZE="`head -1 /proc/meminfo | awk '{ print $2 }'`"
-      # default, without config file overwriding, swap=2*RAM
+      # default, without config file overriding, swap=2*RAM
       CONF_SWAPSIZE="`expr "${MEMSIZE}" '*' "${CONF_SWAPFACTOR}"`"
     fi
 
Index: dphys-swapfile-20061020/dphys-swapfile.8
===================================================================
--- dphys-swapfile-20061020.orig/dphys-swapfile.8	2010-05-05 21:01:59.000000000 +0200
+++ dphys-swapfile-20061020/dphys-swapfile.8	2010-05-05 21:02:06.000000000 +0200
@@ -17,7 +17,7 @@
 .SH DESCRIPTION
 
 dphys-swapfile computes the size for an optimal swap file (and resizes an
-existing swap file if neccessary), mounts an swap file, unmounts it, and and
+existing swap file if necessary), mounts an swap file, unmounts it, and and
 delete it if not wanted any more.
 
 .SH OPTIONS
